framework/src/bundle/CoreBundleContext.cpp

```diff
@@ -250,4 +251,16 @@ namespace cppmicroservices
         }
         return std::string();
     }
+
+    void CoreBundleContext::StopFramework() {
+        this->self.Lock(), stopped = true;
+    }
+
+    void CoreBundleContext::StartFramework() {
+        this->self.Lock(), stopped = false;
+    }
+
+    bool CoreBundleContext::FrameworkStopped() const {
+        return this->self.Lock(), stopped;
```

Comment:
can these functions be re-named to something more accurate and descriptive? The functions only set a flag which indicates the state of the framework.

Reply:
Fixed